diff --git a/arch/arm64/boot/dts/digi/ccimx8qxp-sbc-pro-wb.dts b/arch/arm64/boot/dts/digi/ccimx8qxp-sbc-pro-wb.dts
index a1616c8..04d8dc78 100644
--- a/arch/arm64/boot/dts/digi/ccimx8qxp-sbc-pro-wb.dts
+++ b/arch/arm64/boot/dts/digi/ccimx8qxp-sbc-pro-wb.dts
@@ -98,7 +98,7 @@
  * Conflicts with LPUART3 RX/TX lines.
  */
 &flexcan3 {
-	status = "okay";
+	status = "disable";
 };
 
 &gpu_3d0 {
@@ -275,7 +275,7 @@
  * Conflicts with CAN2.
  */
 &lpuart3 {
-	status = "disabled";
+	status = "okay";
 };
 
 /* PCIE (Wireless) and PCIe (Mini Card) */
